server/ai_mediaserver.go

```diff
@@ -79,6 +79,7 @@ func startAIMediaServer(ls *LivepeerServer) error {
 
 	// This is called by the media server when the stream is ready
 	ls.HTTPMux.Handle("/live/video-to-video/start", ls.StartLiveVideo())
+	ls.HTTPMux.Handle("/live/video-to-video/{stream}/update", ls.StartLiveVideo())
```

Comment:
We could also make the `/start` path above this take the stream  name too, for consistency. (If we change this we should do it soon, before the official release. No real preference though.)

Reply:
I updated to use path everywhere. I _slightly_ prefer it. Let's see what @mjh1 @thomshutt and @victorges say. I'm also ok to keep everywhere the form param.